diem-node/src/lib.rs

```diff
@@ -139,7 +139,8 @@ pub fn load_test_environment<R>(
     template.api.address = template.json_rpc.address;
     template.json_rpc.stream_rpc.enabled = true;
     if lazy {
-        template.consensus.mempool_poll_count = u64::MAX;
+        template.consensus.mempool_poll_count = 30;
```

Comment:
Heads up that I believe this broke our test env print. Will fix in a subsequent PR.

https://github.com/dimroc/diem/blob/shuffle-forge-integration-tests/diem-node/src/lib.rs#L198-L199

cc @zekun000 @xli

Reply:
it's fixed in https://github.com/diem/diem/pull/9732